.github/workflows/publish.yml

```
@@ -0,0 +1,24 @@
+name: npm publish
+
+on:
+  workflow_dispatch:
+
+jobs:
+  npm-publish:
+    name: Publish npm package
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Rust 1.81 # https://github.com/estie-inc/wasm-xlsxwriter/blob/a2696f5e9e729dfe08798309e777cc0e1f3d285b/README.md#browser-support
+        run: rustup install 1.81.0 && rustup default 1.81.0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: "20.x"
+
+      - run: npm ci
+      - run: npm run build
+      - name: Publish
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
      - name: Publish
        run: npm publish --provenance --access public
        env:
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
ref: https://docs.github.com/en/actions/use-cases-and-examples/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry

---

fixed in https://github.com/estie-inc/wasm-xlsxwriter/pull/59/commits/9ffed704d3cbb8a41e4854050e0af275635137bf